Angular/src/app/project/http.service.ts

```diff
@@ -50,6 +50,9 @@ export class HttpService{
 
     updateProject(project: Project) {
         return this.http.put(this.url + '/' + project.id, project);
-    }  
-
+    }
+  
+    getAllProjectTypes(){
+        return this.http.get('https://localhost:5000/api/types');
```

Comment:
put the projectTypeUrl to \app\shared\api-endpoint.constants.ts and get from there